File: 차도희/week4.java
package sep1;

public class n87390 {

	public static void main(String[] args) {
		class Solution {
			
			 public int[] solution(int n, long left, long right) {
				  	int[] answer = new int[(int) (right - left + 1)];
				  	int index = 0;
			        for(long i=left; i <= right; i++)
			        {
			        	int a = (int) i / n;
			        	int b = (int) i % n;
			        	if(a > b)
			        		answer[index] = a + 1;
			        	else
			        		answer[index] = b + 1;
			        	index++;
			        }
			        return answer;
			    }
		}
			
		
		Solution sol = new Solution();
		int[] test = sol.solution(3,2,5);
		for(int i : test) {
			System.out.println(i);
		}

	}

}

File: 이서진/WEEK4.py
def solution(n, left, right):
    array = []
    for index in range(left, right+1):  # 한 행씩
        row = index // n  # /로 하면 float로 출력되어 int()해줘야 하는 이슈 발생해서 //로 변경
        column = index % n  # 나머지
        if row > column:
            array.append(row+1)
        else:
            array.append(column+1)

    return array
